src/app/shared/core/post-capture-card/post-capture-card.component.ts

```diff
@@ -71,4 +92,34 @@ export class PostCaptureCardComponent implements OnInit {
       )
       .subscribe();
   }
+
+  private openCertificate() {
+    zip(this.postCapture$, this.diaBackendAuthService.token$)
+      .pipe(
+        first(),
+        concatMap(([postCapture, token]) =>
+          defer(() => {
+            Browser.open({
+              url: `https://authmedia.net/dia-certificate?mid=${postCapture.id}&token=${token}`,
+            });
+          })
+        ),
+        untilDestroyed(this)
+      )
+      .subscribe();
+  }
+
+  private remove() {
+    return this.postCapture$
+      .pipe(
+        first(),
+        concatMap(postCapture =>
+          this.diaBackendAssetRepository.remove$(postCapture)
+        ),
+        switchTapTo(
+          defer(() => this.diaBackendTransactionRepository.refresh$())
```

Comment:
Refresh `DiaBackendTransactionRepository` will __NOT__ refresh the PostCaptures on the tab. 

You should remove the target PostCapture by modifying the `_postCaptures$` in `post-capture-tab.component.ts` after the deletion, and thus the UI will render without the deleted `PostCaptureCard`.

Please make sure you have tested this behavior is correctly implemented before request the review.